### PR TITLE
added pvacviz entrypoint and basic web server to host client

### PR DESCRIPTION
**Pull request by jmcmichael**
This PR adds a new entrypoint to pvactools, pvacviz, an app that implements a very basic webserver to host the pVACviz client's compiled javascript, css, images, etc. Note that the compiled client isn't included in the repo, instead they'll be added to the pvacviz/client directory and included as part of the pypi package deployment.

Also renamed the pacseq-api endpoint to pvacapi.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -43,17 +43,37 @@
                 filename
             ))
 
+for dirpath, dirnames, filenames in os.walk("utils/pvacviz/client"):
+    for filename in filenames:
+        if not (filename.endswith(".py") or filename.endswith(".pyc")):
+            server_data.append(os.path.join(
+                os.path.relpath(
+                    dirpath,
+                    'utils/pvacapi'
```
Comment (susannasiebert) [inline setup.py]: Change this to `utils/pvacviz/client`.

File: setup.py [hunk 2]
```diff
@@ -43,17 +43,37 @@
                 filename
             ))
 
+for dirpath, dirnames, filenames in os.walk("utils/pvacviz/client"):
```
Comment (susannasiebert) [inline setup.py]: Add a new array `client_data = []` before this line

File: setup.py [hunk 3]
```diff
@@ -43,17 +43,37 @@
...
+    for filename in filenames:
+        if not (filename.endswith(".py") or filename.endswith(".pyc")):
+            server_data.append(os.path.join(
```
Comment (susannasiebert) [inline setup.py]: Append to the `client_data` array instead of `server_data`.

**Comment by susannasiebert** [comment]
I tested your newest commits and it's working for me. Very cool. I noticed that it opens the manage page, not the start page. Is there a reason for one vs the other?

**Comment by jmcmichael** [comment]
Yeah I chose manage as the default, figuring that it would be the most common page user would want to see.

**Pull request merged**